ui/src/app/pages/workspace/workspace-about.spec.tsx

```diff
@@ -176,37 +188,46 @@ describe('WorkspaceAbout', () => {
   });
 
   it('should not display Publish/Unpublish buttons without appropriate Authority', async () => {
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists('[data-test-id="publish-button"]')).toBeFalsy();
-    expect(wrapper.exists('[data-test-id="unpublish-button"]')).toBeFalsy();
-  });
-
-  it('should display Publish/Unpublish buttons with FEATUREDWORKSPACEADMIN Authority', async () => {
-    const profileWithAuth = {
-      ...ProfileStubVariables.PROFILE_STUB,
-      authorities: [Authority.FEATURED_WORKSPACE_ADMIN],
-    };
-    profileStore.set({ profile: profileWithAuth, load, reload, updateCache });
-
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists('[data-test-id="publish-button"]')).toBeTruthy();
-    expect(wrapper.exists('[data-test-id="unpublish-button"]')).toBeTruthy();
+    componentAlt();
+    await waitForNoSpinner();
+    expect(
+      screen.queryByRole('button', {
+        name: 'Publish',
+      })
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', {
+        name: 'Publish',
+      })
+    ).not.toBeInTheDocument();
   });
 
-  it('should display Publish/Unpublish buttons with DEVELOPER Authority', async () => {
-    const profileWithAuth = {
-      ...ProfileStubVariables.PROFILE_STUB,
-      authorities: [Authority.DEVELOPER],
-    };
-    profileStore.set({ profile: profileWithAuth, load, reload, updateCache });
-
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.exists('[data-test-id="publish-button"]')).toBeTruthy();
-    expect(wrapper.exists('[data-test-id="unpublish-button"]')).toBeTruthy();
-  });
+  test.each([
+    ['FEATUREDWORKSPACEADMIN', Authority.FEATURED_WORKSPACE_ADMIN],
+    ['DEVELOPER', Authority.DEVELOPER],
+  ])(
+    `should display Publish/Unpublish buttons with %s Authority`,
+    async (_, authority) => {
+      const profileWithAuth = {
+        ...ProfileStubVariables.PROFILE_STUB,
+        authorities: [authority],
+      };
+      profileStore.set({ profile: profileWithAuth, load, reload, updateCache });
+
+      componentAlt();
+      await waitForNoSpinner();
+      expect(
+        screen.getByRole('button', {
+          name: 'Publish',
+        })
+      ).toBeInTheDocument();
+      expect(
+        screen.getByRole('button', {
+          name: 'Unpublish',
+        })
+      ).toBeInTheDocument();
+    }
+  );
 
   it('Publish/Unpublish button styling depends on state - unpublished', async () => {
```

Comment:
I am not sure about these tests. Enzyme had access to the higher level component so it could get the `type` value. The `type` value is not available on the DOM. In order to keep this test, I feel like we can either compare the style of the button to the style of our primary/secondary button styles or we can add something like a `data-button-type` attribute to our button component.

I went with the `data-button-type` approach after realizing that the styling approach seemed messy and brittle. I don't love the `data-button-type` approach either, but it seems to be the better of the two.

Thoughts?